src/lang/modifyAst.ts

```diff
@@ -1345,15 +1169,15 @@ export function createNodeFromExprSnippet(
 export function insertVariableAndOffsetPathToNode(
   variable: KclCommandValue,
   modifiedAst: Node<Program>,
-  pathToNode: PathToNode
+  pathToNode?: PathToNode
```

Comment:
@franknoirot @Irev-Dev this way I can use it in `addSweep.ts`. Doesn't seem like a big change since we had the typeof check already